scripts/publish_posts.py

```
@@ -5,6 +5,7 @@
 import yaml
 import re
 import logging
+from typing import List, Dict, Optional, Tuple
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)